src/models/TalkMapper.php

```diff
@@ -784,4 +784,16 @@ public function getSpeakerFromTalk($talk_id, $display_name)
         return count($speakers) > 0 ? $speakers[0] : false;
 
     }
+
+    public function removeSpeakerFromTalk($talk_id, $speaker_id)
+    {
+        $params = [
+            'talk_id' => $talk_id,
+            'speaker_id' => $speaker_id,
+        ];
+
+        $sql = 'update talk_speaker set speaker_id = null where talk_id = :talk_id and speaker_id = :speaker_id';
```

Comment:
Am I right in thinking that this removal only works when a speaker has claimed the talk and the claim was approved? So removing a speaker that has not yet claimed the talk doesn't work currently?